examples/src/main/scala/org/apache/spark/examples/sql/SQLDataSourceExample.scala

```diff
@@ -246,6 +247,47 @@ object SQLDataSourceExample {
     // +---------------+----+
     // $example off:json_dataset$
   }
+  
+  private def runCsvDatasetExample(spark: SparkSession): Unit = {
+    // $example on:csv_dataset$
+    val path = "examples/src/main/resources/people.csv"
+
+    val df = spark.read.csv(path)
+    df.show()
+    // +------------------+
+    // |               _c0|
+    // +------------------+
+    // |      name;age;job|
+    // |Jorge;30;Developer|
+    // |  Bob;32;Developer|
+    // +------------------+
+
+    // Read a csv with delimiter, the default delimiter is ","
+    val df2 = spark.read.options(Map("delimiter"->";")).csv(path)
+    df2.show()
+    // +-----+---+---------+
+    // |  _c0|_c1|      _c2|
+    // +-----+---+---------+
+    // | name|age|      job|
+    // |Jorge| 30|Developer|
+    // |  Bob| 32|Developer|
+    // +-----+---+---------+
+
+    // Read a csv with delimiter and a header
+    val df3 = spark.read.options(Map("delimiter"->";","header"->"true")).csv(path)
```

Comment:
Can you use `option` instead of `options(Map(...))` to be consistent with other examples?

Reply:
I add `.option()` and `.options()` to both Scala and Python examples.